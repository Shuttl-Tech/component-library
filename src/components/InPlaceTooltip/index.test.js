import React from 'react';
import renderer from 'react-test-renderer';

import { InPlaceTooltip } from './';

const tooltipText = 'Hello World!';

it('InPlaceTooltip tree looks like how it should', () => {
	let props = {
		position: 'left'
	};

	const component = renderer.create(<InPlaceTooltip {...props}>{tooltipText}</InPlaceTooltip>);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
