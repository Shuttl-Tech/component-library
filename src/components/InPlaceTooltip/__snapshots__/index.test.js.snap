// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InPlaceTooltip tree looks like how it should 1`] = `
<div
  className="tooltip tooltip--left component--in-place-tooltip component--in-place-tooltip--left"
>
  Hello World!
</div>
`;
